test/unit/browser/cache/cache_test.dart

```diff
@@ -204,31 +225,37 @@ void main() {
         var childCache = new Cache(stubCachingStrategy);
         await childCache.get(cachedId, () => cachedValue);
         await childCache.remove(cachedId);
-        verify(stubCachingStrategy.onDidRemove(cachedId, cachedValue));
+
+        expect(stubCachingStrategy.onDidRemoveCalled == 1, isTrue);
```

Comment:
Better error messages if you do `expect(stubCachingStrategy.onDidRemoveCalled, 1);`, same goes for the others.